lang/test/org/partiql/lang/eval/EvaluatingCompilerDateTimeTests.kt

```diff
@@ -56,12 +56,13 @@ class EvaluatingCompilerDateTimeTests : EvaluatorTestBase() {
         assertEquals(ion.newInt(expectedTime.tz_minutes?.rem(MINUTES_PER_HOUR)), actual["timezone_minute"])
     }
 
-    data class TimeTestCase(val query: String, val expected: String, val expectedTime: TimeForValidation? = null)
+    data class TimeTestCase(val query: String, val expected: String, val expectedTime: TimeForValidation? = null, val session: EvaluationSession = EvaluationSession.standard())
 
     @ParameterizedTest
     @ArgumentsSource(ArgumentsForTimeLiterals::class)
     fun testTime(tc: TimeTestCase)  {
-        val originalExprValue = eval(tc.query)
+        val originalExprValue = eval(source = tc.query,
+                                     session = tc.session)
```

Comment:
nit: could put on one line:
```suggestion
        val originalExprValue = eval(source = tc.query, session = tc.session)
```